### PR TITLE
Release 1.1.facilities mc2

### PR DESCRIPTION
**Pull request by delarosatrevin**
Changes in this PR:

* Updated version of motioncor2-1.0.1
* Removed conversion from tif for motioncor2

Slightly more general way to express that input movies need to be converted to a given format.

### CODE REVIEW

**Comment by azazellochg** [comment]
Hi @delarosatrevin I'm afraid I have no time to properly review it during next two weeks, could you ask someone else?

**Comment by pconesa** [comment]
We will here, thank you both!

**Comment by delarosatrevin** [comment]
No problem @azazellochg I ask you first because you have integrated this protocol.
@josuegbl Can you review this PR?

**Comment by josuegbl** [comment]
Yes, I can....

File: pyworkflow/em/packages/motioncorr/protocol_motioncorr.py [hunk 1]
```diff
@@ -139,6 +154,15 @@ def _defineAlignmentParams(self, form):
...
         line.addParam('patchY', params.IntParam, default=5, label='Y')
 
+        if self.isLatestVersion():
+            form.addParam('patchOverlap', params.IntParam, default=0,
+                          label='Patches overlap (%)',
+                          help="Specify the overlapping between adjacent "
+                               "patches. \nFor example, overlap=20 means that "
+                               "each patch will have a 20% overlapping with "
+                               "its neighboring patches in each dimension. \n"
+                               "Note: NEW in version 1.0.1")
```
[the remark this answers is not in the document]
Comment (delarosatrevin) [reply pyworkflow/em/packages/motioncorr/protocol_motioncorr.py]: Nop, it is true that this feature was added in version 1.0.1, even if we support 1.0.2 :P

File: pyworkflow/em/packages/motioncorr/convert.py [hunk 2]
```diff
@@ -93,7 +93,8 @@ def getSupportedVersions(var):
     if var == 'MOTIONCORR':
         return ['2.1']
     elif var == 'MOTIONCOR2':
-        return ['03162016', '10192016', '01302017', '1.0.0']
+        return ['03162016', '10192016', '01302017',
```
[the remark this answers is not in the document]
Comment (delarosatrevin) [reply pyworkflow/em/packages/motioncorr/convert.py]: done

File: pyworkflow/em/packages/motioncorr/protocol_motioncorr.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (josuegbl) [reply pyworkflow/em/packages/motioncorr/protocol_motioncorr.py]: 👍

**Pull request merged**